tests/resources/module_with_spaced_year_range_in_license.py

```diff
@@ -0,0 +1,6 @@
+# Copyright (C) 2015 - 2017 Teela O'Malley
```

Comment:
Could you also please add a unit test for this fix:

> Stop changing four digit numbers that come after the header.

Reply:
This is handled in the module_with_license_and_numbers.py file.

Reply:
Oh, alright. I was expecting a similar `"2015 - 2017"` string to appear in the code